pkg/cmd/auth/login/login.go

```diff
@@ -59,6 +59,10 @@ func NewCmdLogin(f *cmdutil.Factory, runF func(*LoginOptions) error) *cobra.Comm
 		Long: heredoc.Docf(`
 			Authenticate with a GitHub host.
 
+			Your %[1]shostname%[1]s is the domain where the GitHub instance you are logging into is hosted.
+			For example, "github.com" is the default hostname for GitHub.com. To authenticate with
+			a different hostname like github.enterprise.com, use that hostname instead.
+
```

Comment:
Looking for feedback, here 🙂

Reply:
There's some weird or awkward about this description and I wonder if it's because of the term `hostname`. Technically, `hostname` can be part of a domain. Here's what I'm thinking:

```suggestion
			The %[1]shost%[1]s is the domain where the GitHub instance you are logging into is hosted.
			For example, "github.com" is the where GitHub is hosted. To authenticate with
			a different host like github.enterprise.com, enter that domain instead.

```

Reply:
Seeing the rest of the PR, I realize that my suggestion would also change the scope a bit since `hostname` already existed as a prompt. 😅 In that case, maybe:

```suggestion
			The %[1]shostname%[1]s is the domain where the GitHub instance you are logging into is hosted.
			For example, "github.com" is the hostname for GitHub.com. To authenticate with
			a different hostname like github.enterprise.com, use that hostname instead.

```

Reply:
This seems very verbose. I think we could keep it simpler. If we want to give examples, we should update/add examples to the examples section around L89. We already have an example for using `--hostname` but it could do with a revision maybe:

```
EXAMPLES
  # Start interactive setup
  $ gh auth login

  # Authenticate against github.com by reading the token from a file
  $ gh auth login --with-token < mytoken.txt

  # Authenticate with specific host
  $ gh auth login --hostname enterprise.internal
```

Anyways, here's my suggestion for this part:

```suggestion
			Your %[1]shostname%[1]s is where you login to GitHub. The default hostname is GitHub.com.

```

Reply:
I do like simpler... Do you think any folks will get confused without the example?

Reply:
Personally, I don't think anyone will be confused without the example. Folks needing to provide a hostname, such as GHES users, should already be familiar with the idea of providing a hostname that isn't GitHub.com for interacting with their appliance. 

There's also that example that already exists for `--hostname` usage lower in the doc. If anything, I think improving that example could be worthwhile.

But if it was my decision, I'd just keep it simple for now :)

Reply:
That's compelling. I've shortened it to your change, here. I think the example down below in the code clears up anything that might be unclear with this description. Thanks for the feedback!